.travis.yml

```diff
@@ -191,21 +191,20 @@ before_install:
           && ALPAKA_CUDA_VER_MINOR=${ALPAKA_CUDA_VER:2:1}
           && echo ${ALPAKA_CUDA_VER_MINOR}
           && ALPAKA_CUDA_CACHE_DIR=${HOME}/cache/CUDA/CUDA-${ALPAKA_CUDA_VER}
-          && if [ ${ALPAKA_CUDA_VER_MAJOR} == 7 ]
+          && if (( (( ${ALPAKA_CUDA_VER_MAJOR} == 7 )) && (( ${ALPAKA_CUDA_VER_MINOR} == 0 )) ))
```

Comment:
I am no bash pro and so I do not know the specific reasons. 😄 It just works. If there is a better syntax, please create a Pull Request.